docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -270,47 +270,39 @@ PUT _ingest/pipeline/tri_met_tracks
 ----------------------------------
 
 [float]
-==== Step 2: Start {filebeat}
+==== Step 2: Configure {agent}
 
-. Replace the contents in your `filebeat.yml` file with the following:
-+
-[source,yaml]
-----------------------------------
-filebeat.inputs:
-# Fetch trimet bus data every minute.
-- type: httpjson
-  interval: 1m
-  request.url: "https://developer.trimet.org/ws/v2/vehicles?appID=<tri_met_app_id>"
-  response.split:
-    target: body.resultSet.vehicle
-  processors:
-    - decode_json_fields:
-        fields: ["message"]
-        target: "trimet"
+. From the {kib} main menu, click *Fleet*, then the *Agent policies* tab.
 
-  pipeline: "tri_met_tracks"
+. Click the name of the agent policy where you want to add the *Custom API* integration. The configuration changes you make only apply to the policy you select.
 
+. Click the name of the *Custom API* integration, or add the integration if the agent policy does not yet have it.
 
-# ---------------------------- Elastic Cloud Output ----------------------------
-cloud.id: <cloud_id>
-cloud.auth: <username:password>
+. From the *Edit Custom API integration* page, expand the *Change defaults* section.
 
-----------------------------------
+. Set the *Dataset name* to *httpjson.trimet*.
```

Comment:
Maybe I don't understand Fleet very well, but isn't this the name of the index (e.g. `tri_met_tracks`)?